elixir/wordcount.ex

```diff
@@ -0,0 +1,29 @@
+IO.stream(:stdio, :line)
+# Create a dictionary in which the keys are the words and the values are
+# the number of appearances in the input text.
+|> Enum.reduce(%{}, fn (line, acc) ->
+  line
+  |> String.strip
+  |> String.split(~r/\s/)
```

Comment:
You should omit String.split and pass split: true, will probably make it a little bit faster.